docs/src/tutorials/send-events.md

```diff
@@ -0,0 +1,56 @@
+# Thin Edge Event
+
+Events on thin-edge.io can be used to trigger signals when some event happens in the system.
+For example, a person entering a room or someone logging into a machine/website can all be represnted as events.
+Events are stateless and hence are processed as and when they occur.
+They don't represent state but can be used to represent state changes.
+An event can't be updated/cleared once its triggered, unlike alarms that are cleared explicitly after processing.
+
+Every alarm is uniquely identified by its type.
+If multiple events are raised for a given type, thin-edge.io will process them all separately in the order in which they were raised.
+
+## Sending an event
+
+An event can be triggered on thin-edge.io by sending an MQTT message in Thin Edge JSON format to certain MQTT topics.
+
+The scheme of the topic to publish the event data is as follows:
+
+`tedge/events/<event-type>`
+
+The payload format must be as follows:
+
+```json
+{
+    "message": "<message text>",
+    "time": "<Timestamp in ISO-8601 format>"
+}
+```
+
+Here is a sample event triggered for a `login_event` event type:
+
+Topic: 
+`tedge/events/login_event`
+
+Payload:
+```json
+{
+    "message": "A user just logged in",
+    "time": ""2021-01-01T05:30:45+00:00""
+}
+```
+
+> Note: Both the `message` field and the `time` field are optional.
+When a `message` is not provided, a placeholder message like `generic event` would be generated if the connected cloud mandates one.
```

Comment:
There should probably be a blank line before this paragraph, syntax highlighting in github detects this as quote because the line above is a quote. Do you care to change this?